### DISCUSSION TITLE
What distro is recommended for building a self-hosted Supabase instance?

### DISCUSSION THREAD

**Discussion by MrRedBeard** [supabase/supabase#455] [Questions] [answered]
I have tried the docker and node/docker setup on RancherOS and Ubuntu with meh results so I want to attempt to do this manually. 

What linux distro is recommended for running supabase?

**Comment by kiwicopple** [comment, marked as answer]
Hi @MrRedBeard 

> tried the docker and node/docker

Which setup were you using? We haven't really put much energy into the self-hosted setup yet since we are still building so many features. Internally we are using Ubuntu though
